nixd/lib/Controller/LifeTime.cpp

```diff
@@ -34,6 +34,19 @@ opt<std::string> DefaultNixOSOptionsExpr{
          "=  (import <nixpkgs/nixos/modules/module-list.nix>) ++ [ ({...}: { "
          "nixpkgs.hostPlatform = builtins.currentSystem;} ) ] ; })).options")};
 
+opt<std::string> DefaultDevenvOptionsExpr{
+    "devenv-options-expr",
+    desc("Default expression interpreted as devenv option declarations"),
+    cat(NixdCategory),
+    init("let\n"
+         "  currentSystem = builtins.currentSystem;\n"
+         "  flake = builtins.getFlake \"github:cachix/devenv?ref=work/keys/expose-devenv-options\";\n"
```

Comment:
Connecting to a remote flake may initially take some time due to its initial setup process. Additionally, it's likely that the URL will need to be updated once the changes from this pull request (https://github.com/cachix/devenv/pull/1390) have been merged.

I'd like to [open a request window](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#window_showMessageRequest) here, maybe asking the user whether or not they want to let us fetch it.

Reply:
@inclyc Thank you for the review, regarding the `open a request window` I don't see that being implemented in `nixd`, is it the case. If not I can try to implement it : )

Reply:
> I don't see that being implemented in `nixd`, is it the case. If not I can try to implement it : )

Not done yet, we can do this in subsequent patches.